internal/mockserver/mockdata/inventories.go

```diff
@@ -204,4 +204,204 @@ var (
+            "update_inventory_sources": "/api/v2/inventories/6/update_inventory_sources/",
+            "inventory_sources": "/api/v2/inventories/6/inventory_sources/",
+            "organization": "/api/v2/organizations/1/"
+        },
+        "summary_fields": {
+            "organization": {
+                "id": 1,
+                "name": "Default",
+                "description": ""
+            },
+            "created_by": {
+                "id": 1,
+                "username": "admin",
+                "first_name": "",
+                "last_name": ""
+            },
+            "modified_by": {
+                "id": 1,
+                "username": "admin",
+                "first_name": "",
+                "last_name": ""
+            },
+            "object_roles": {
+                "use_role": {
+                    "id": 80,
+                    "description": "Can use the inventory in a job template",
+                    "name": "Use"
+                },
+                "admin_role": {
+                    "id": 78,
+                    "description": "Can manage all aspects of the inventory",
+                    "name": "Admin"
+                },
+                "adhoc_role": {
+                    "id": 77,
+                    "description": "May run ad hoc commands on an inventory",
+                    "name": "Ad Hoc"
+                },
+                "update_role": {
+                    "id": 81,
+                    "description": "May update project or inventory or group using the configured source update system",
+                    "name": "Update"
+                },
+                "read_role": {
+                    "id": 79,
+                    "description": "May view settings for the inventory",
+                    "name": "Read"
+                }
+            },
+            "user_capabilities": {
+                "edit": true,
+                "copy": true,
+                "adhoc": true,
+                "delete": true
+            }
+        },
+        "created": "2018-08-13T01:59:47.160127Z",
+        "modified": "2018-08-13T01:59:47.160140Z",
+        "name": "TestInventory-update1",
+        "description": "for testing UpdateInventory api",
+        "organization": 1,
+        "kind": "",
+        "host_filter": null,
+        "variables": "",
+        "has_active_failures": false,
+        "total_hosts": 0,
+        "hosts_with_active_failures": 0,
+        "total_groups": 0,
+        "groups_with_active_failures": 0,
+        "has_inventory_sources": false,
+        "total_inventory_sources": 0,
+        "inventory_sources_with_failures": 0,
+        "insights_credential": null,
+        "pending_deletion": false
+    }`)
+
+	// MockedDeleteInventoryResponse mocked `Delete` endpoint response
+	MockedDeleteInventoryResponse = []byte(`{}`)
+
+	// MockedGetInventoryByIDResponse mocked `GetInventoryByID` endpoint response
+	MockedGetInventoryByIDResponse = []byte(`
```

Comment:
In order to keep same code style, pls rename `MockedGetInventoryByIDResponse` to `MockedGetInventoryResponse` also the comment part.